pkgs/development/beam-modules/build-mix.nix

```diff
@@ -0,0 +1,81 @@
+{ stdenv, writeText, elixir, erlang, hex, lib }:
+
+{ name
+, version
+, src
+, buildInputs ? []
+, beamDeps ? []
```

Comment:
I would tend to add a comment here.
`Called beamDeps, because it's a mix of elixir and erlang dependencies.`
This is just nitpick, feel free to ignore